test/features/deploy_springboot_service.feature

```diff
@@ -109,3 +111,64 @@ Feature: Deploy spring boot service
 
     Then Service "process-springboot-example" contains 0 instance of process with name "orders"
     And Service "process-springboot-example" contains 0 instance of process with name "orderItems"
+
+#####
+
+  @externalcomponent
+  @infinispan
+  @persistence
+  Scenario: Deploy process-springboot-example service with persistence using external Infinispan
+    Given Kogito Operator is deployed with Infinispan operator
+    And Infinispan instance "external-infinispan" is deployed with configuration:
+      | username | developer |
+      | password | mypass    |
+    And Deploy springboot example service "process-springboot-example" with configuration:
+      | infinispan | username    | developer                 |
+      | infinispan | password    | mypass                    |
+      | infinispan | uri         | external-infinispan:11222 |
+    And Kogito application "process-springboot-example" has 1 pods running within 10 minutes
+    And Start "orders" process on service "process-springboot-example" within 3 minutes with body:
+      """json
+      {
+        "approver" : "john", 
+        "order" : {
+          "orderNumber" : "12345", 
+          "shipped" : false
+        }
+      }
+      """
+    And Service "process-springboot-example" contains 1 instances of process with name "orders"
+
+    When Scale Kogito application "process-springboot-example" to 0 pods within 2 minutes
+    And Scale Kogito application "process-springboot-example" to 1 pods within 2 minutes
+
+    Then Service "process-springboot-example" contains 1 instances of process with name "orders" within 2 minutes
+    # Native application is not tested as the test is very expensive and the external Infinispan configuration shouldn't have any effect on build type
```

Comment:
I guess this comment does not apply to a springboot app.

Reply:
Good catch, it is a copy/paste leftover